installer/build-mac-dmg.sh

```diff
@@ -88,23 +88,28 @@ while [ "$(( $(date +%s) - 3600 ))" -lt "$START" ]; do
     notarize_info=$(xcrun altool --notarization-info "$REQUEST_UUID" -u "jonathan@openshot.org" -p "@keychain:NOTARIZE_AUTH")
     echo "$notarize_info"
 
-    pat='Status: (.*)'
```

Comment:
`.` matches all new lines after `Status: ` as well... and due to Apple changing the response syntax... this started breaking. For example, $notarize_status would equal:

```
in progress\n'
b'Status Code: 0\n
```